src/towncrier/test/test_check.py

```diff
@@ -148,3 +172,52 @@ def test_none_stdout_encoding_works(self):
 
         self.assertEqual(0, proc.returncode)
         self.assertEqual(b"", stderr)
+
+    def test_first_release(self):
+        runner = CliRunner()
+
+        with runner.isolated_filesystem():
+            # Arrange
+            write("towncrier.toml", "[tool.towncrier]")
+            write("newsfragments/.gitignore", "!.gitignore")
+            initial_commit()
+
+            write("newsfragments/123.feature", "Foo the bar")
+            commit("Foo the bar")
+
+            call(["git", "checkout", "-b", "next-resease"])
+            call(["towncrier", "--yes", "--version", "1.0"])
+            commit("Prepare a release")
+
+            # Act
+            result = runner.invoke(_main, ["--compare-with", "main"])
+
+            # Assert
+            self.assertEqual(0, result.exit_code, (result, result.output))
+            self.assertIn("Checks SKIPPED: news file changes detected", result.output)
+
+    def test_second_release(self):
```

Comment:
I don't understand this test. What code path / feature is tested here that is not tested in the first test?